custom_components/zaptec/__init__.py

```diff
@@ -69,6 +69,14 @@
     Platform.UPDATE,
 ]
 
+CHECK_ENTITY_AVAILABLE_ON_ADD = False
+"""Enable a check to ensure that the entity is available when added.
+
+If not available, the entity will not be added to the list of entities.
+For Zaptec, this is not recommended, as some attributes are not always
+available. E.g. "TotalChargePowerSession" is only present under certain
+conditions.
```

Comment:
Does "some" mean 1-2 or significantly more? In other words: do we want to disable this for all entities, or is there a limited number this applies to?

Reply:
Even if we don't add them, they are not removed from the entity registry, right?

So if we have a list of the entities we want to skip the check for in the first 8.0-beta, we can include instructions for beta testers to look out for these entities "no longer being provided by the zaptec integration." and the entities will return safely after an updated beta with an extended list?

Reply:
So could something like `if description.key not in KEYS_TO_SKIP_ENTITY_AVAILABILITY_CHECK_FOR` where `KEYS_TO_SKIP_ENTITY_AVAILABILITY_CHECK_FOR = ["total_charge_power_session"]` work?